react/components/LocalQuery.js

```diff
@@ -9,6 +9,46 @@ import { SORT_OPTIONS } from '../OrderBy'
 const DEFAULT_PAGE = 1
 const DEFAULT_MAX_ITEMS_PER_PAGE = 10
 
+const QUERY_SEPARATOR = '/'
+const MAP_SEPARATOR = ','
+
+const splitQuery = split(QUERY_SEPARATOR)
+const splitMap = split(MAP_SEPARATOR)
+const joinQuery = join(QUERY_SEPARATOR)
+const joinMap = join(MAP_SEPARATOR)
+
+const includeFacets = (map, query) =>
+  !!(map && map.length > 0 && query && query.length > 0)
+
+const useFacetsArgs = (query, map) => {
+  return useMemo(() => {
+    const queryArray = splitQuery(query)
+    const mapArray = splitMap(map)
+    const queryAndMap = zip(queryArray, mapArray)
+    const relevantArgs = [
+      head(queryAndMap),
+      ...tail(queryAndMap).filter(
+        ([_, tupleMap]) => tupleMap === 'c' || tupleMap === 'ft'
+      ),
+    ]
+    const { finalMap, finalQuery } = relevantArgs.reduce(
+      (accumulator, [tupleQuery, tupleMap]) => {
+        accumulator.finalQuery.push(tupleQuery)
```

Comment:
kinda weird to use `reduce` here and mutate this at the same time. This could be a `forEach`, and mutate an external var (or destructure the accumulator on each iteration, on the other hand). but w/e

Reply:
ja foi deployado assim na store, não vou mudar por enquanto mas prometo que numa pr próxima ja melhoro isso, valeu pela observação e concordo